README.md

```diff
@@ -5,3 +5,25 @@ This repository is the primary reference for the Clarity language.
 Note that this reference is not normative. It may include details that are specific to the Stacks blockchain implementation of Clarity itself, and should not be taken as a specification for the Clarity language. Such a specification may come someday, but this is what we have for now.
 
 You can read [the language reference](https://github.com/clarity-lang/reference/blob/master/reference.md).
+
+# Contributing
+
+Anyone can open issues on this repository to suggest improvements to the Clarity language.
+The point is to have early discussions here before moving to an actual SIP.
+
+- Make sure that a similar issue doesn't already exists (check the closed ones)
+- Be specific
+- Give use cases and usage exemples
+
+## From candidate to SIP
+
+Once an issue is opened, it can have the tag `clarity-3-candidate`, meaning that the idea will be considered to be added to the upcoming version clarity.
```

Comment:
```suggestion
Once an issue is opened, it can have the tag `clarity-3-candidate`, meaning that the idea will be considered to be added to the upcoming version of Clarity.
```